spec/support/github_service_helper.rb

```diff
@@ -0,0 +1,207 @@
+require 'octokit'
+require 'github_service'
+require 'github_service/issue'
+
+module GithubServiceHelper
+  module_function
+
+  # Helper method for configuring the stubs for the GithubService's @service
+  # (Octokit::Client) variable.
+  #
+  # Can be configured in the rspec instance as follows:
+  #
+  #   github_service_stubs do |stub|
+  #     # stub getting issue 1234
+  #     issue_url = '/repos/ManageIQ/manageiq-cross_repo-tests/issues/1234'
+  #     stub.get(issues_url, default_octokit_headers) do |env|
+  #        [200, github_service_response_headers, '{}']
+  #     end
+  #   end
+  #
+  #   # stub creating a new PR on ManageIQ/manageiq-cross_repo-tests
+  #   new_pr_url = '/repos/ManageIQ/manageiq-cross_repo-tests/pulls'
+  #   github_service_stubs.post(new_pr_url, nil, headers) do |env|
+  #      [
+  #        200,                              # response status
+  #        github_service_response_headers,  # response headers
+  #        {                                 # response body
+  #          "id" => 1,
+  #          # ...
+  #        }.to_json
+  #      ]
+  #   end
+  #
+  def github_service_stubs(&block)
```

Comment:
Why are we adding these stubs?  I'd rather we didn't and for this case I don't see why we need it.

Reply:
FYI, this was merged in another PR.